_summaries/kingma2014adam_1.md

```diff
@@ -3,17 +3,39 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jordi1215
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Adam (derived from adaptive moment estimation) is an optimization algorithm that can be used instead of the classical stochastic gradient descent procedure to update network weights iterative based in training data.
```

Comment:
I suggest pointing out the core motivation of the paper, i.e., the problem of existed methods and why are they important. Something like: SFO is not memory efficient, or RMS does not have a bias-correction term.